File: system/tests/rspec/pages/utilities/sql_manager.rb
class SqlManager < ControlPanelPage

	element :table, 'table'
	elements :tables, 'table tr td:first-child'
	elements :manage_links, 'td li.view a'
	element :search_field, 'input[name=search]'
	element :search_btn, 'input[name=search_form]'
	element :op_select, 'select[name=table_action]'
	element :op_submit, '.tbl-bulk-act input[type=submit]'

	def load
		self.open_dev_menu
		click_link 'Utilities'
		click_link 'SQL Manager'
	end
end

File: tests/cypress/cypress/elements/pages/utilities/Sql_manager.js
import ControlPanel from '../ControlPanel'

class SqlManager extends ControlPanel {
    constructor() {
        super()


        this.selectors = Object.assign(this.selectors, {
		  "table": 'table',
		  "tables": 'table tr td:first-child',
		  "manage_links": 'td li.view a',
		  "search_field": 'input[type!=hidden][name=search]',
		  "search_btn": 'input[type!=hidden][name=search_form]',
		  "op_select": 'select[name=bulk_action]',
		  "op_submit": '.bulk-action-bar [type=submit]'
        })

    }

     get_tables(){
    	return cy.task('db:query', 'SHOW TABLES').then(function([rows,fields]){

    		return rows;
    	})
    }

    load(){
    	this.open_dev_menu()
	    cy.get('body > .dropdown').find('a:contains("Utilities")').click()
	    this.get('wrap').find('a:contains("SQL Manager")').click()

    }




}
export default SqlManager